nautobot/extras/models/jobs.py

```diff
@@ -804,6 +810,9 @@ def clean(self):
         # bitwise xor also works on booleans, but not on complex values
         if bool(self.approved_by_user) ^ bool(self.approved_at):
             raise ValidationError("Approval by user and approval time must either both be set or both be undefined")
+        if self.interval == JobExecutionType.TYPE_CUSTOM:
+            if self.get_crontab() is str:
+                raise ValidationError(self.get_crontab())
```

Comment:
This pattern seems weird to be evaluating the return value of the method using an identity check (`is str`) and then calling the same method again to raise its return value as an exception. If it's expected that `get_crontab()` should return an error, it would be best to just let that error occur and expect the callers to handle it.

It also appears that you're not actually doing anything with the result of `get_crontab()` in the event that it doesn't raise an error.

So IMO if we had `get_crontab()` just raise a `ValidationError` then you'd only have to call `get_crontab()` here to assert that the crontab is accurate or raises an error.

Reply:
removing this while also removing the try except in the get_crontab() method should resolve